tests/unit/lms/services/async_oauth_http_test.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+from unittest.mock import sentinel
+
+import httpretty
+import pytest
+from aiohttp import TooManyRedirects
+from aioresponses import CallbackResult, aioresponses
+
+from lms.services import ExternalRequestError
+from lms.services.async_oauth_http import AsyncOAuthHTTPService, factory
+
+
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
+class TestAsyncOAuthHTTPService:
+    def test_get_success(
+        self, svc, urls, with_successful_responses, oauth2_token_service
+    ):
+        responses = svc.get(urls)
+
+        for i, url in enumerate(urls):
+            assert responses[i].status == 200
+
+            # Order is kept from the URLs param
+            assert responses[i].headers["url"] == url
```

Comment:
You can use `zip()`:

```suggestion
        for url, response in zip(urls, responses):
            assert response.status == 200
            assert response.headers["url"] == url
```